frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTableRow.tsx

```diff
@@ -108,7 +111,6 @@ const PipelineRunTableRow: React.FC<PipelineRunTableRowProps> = ({
       <Td dataLabel="Name">
         <PipelineRunTableRowTitle run={run} />
       </Td>
-      <Td dataLabel="Experiment">{experiment?.display_name || 'Default'}</Td>
```

Comment:
Would we not need this when we have the run page under Pipelines?

Reply:
Yea, I missed this as I was mainly testing on the Experiments page. Added back.